website/pages/docs/nia/tasks.mdx

```diff
@@ -8,4 +8,24 @@ description: >-
 
 # Tasks
 
-A task is the translation of dynamic service information from the Consul Catalog into network infrastructure changes downstream. Consul-Terraform-Sync carries out automation for executing tasks using network drivers. The scope of a task in terms of the Terraform driver is the execution of a Terraform module. An example of a task is to automate a firewall security policy rule with discovered IP addresses for a set of Consul services.
+A task is the translation of dynamic service information from the Consul Catalog into network infrastructure changes downstream. Consul-Terraform-Sync carries out automation for executing tasks using network drivers. For a Terraform driver, the scope of a task is a Terraform module.
+
+Below is an example task configuration:
+
+```shell-session
```

Comment:
Changing shell-session to HCL will get the nice syntax coloring 😄

Reply:
thanks! it looks pretty now